lib/audited/auditor.rb

```diff
@@ -230,8 +231,13 @@ def revision_with(attributes)
 
       private
 
-      def audited_changes
-        all_changes = respond_to?(:changes_to_save) ? changes_to_save : changes
+      def audited_changes(for_touch: false)
+        all_changes = if for_touch
+                        previous_changes
+                      else
+                        respond_to?(:changes_to_save) ? changes_to_save : changes
+                      end
```

Comment:
This seems a bit clearer to me, but 🤷 
```suggestion
        all_changes = if for_touch
                        previous_changes
                      elsif respond_to?(:changes_to_save)
                        changes_to_save
                      else
                        changes
                      end
```